### ISSUE TITLE
'Invalid Account' when logging in with new Admin user.

### ISSUE THREAD

**Issue by thanateros** [qor/qor-example#120]
Fresh pull of qor-example this morning, configured config/database.yml, manually created the qor_example database, ran the seeds program per the instructions to populate the database.  

Website looks fine. I log into Admin Dashboard with dev@getqor.com/testing with no problem.

My problem is when I create a new user with role 'Admin' via the Admin interface I cannot then log in with that user; when I try I get a red flash with 'Invalid Account'.

What am I doing wrong?

**Comment by cryptix** [comment]
Hmm.. I usually don't use the seeding and the first user I then create gets role Admin by default (would actually like to know where this is handled but didn't check yet).

Otherwise I tend to use the logging mailer during configuration/setup and just reset the password using the reset link.

**Comment by alexmironof** [comment]
@thanateros Try to create user from frontend and then set him admin role from administration interface.
I guess there something wrong with passwords in admin(because new auth arrives ricently)

**Comment by thanateros** [comment]
Hey thanks guys; I forgot I still had this open.  On this particular issue I just registered a new user and flipped it to admin in the database.  Since then I started a new project from scratch using qor-example as a guide.  Still never figured out why qor-example wouldn't let me log in with a new account in that build; however, that doesn't seem to be a problem with commits after the one I was using.
